README.md

```diff
@@ -75,27 +75,66 @@ Activate the virtual environment in Python
 
 `deactivate` to exit it from venv
 
-## `pip install -r requirement.txt`
+## `pip install -r requirements.txt`
 
 It will looks for the package in PyPI, resolves its dependencies, and installs everything in your current Python environment to ensure that requests will work.
 
 ## `pip freeze > requirements.txt`
 
-It records an environment's current package list into requirements. txt
-
-## `pip install uvicorn`
-
-Runs `pip install uvicorn` if you do not have uvicorn already as it is required.
+It records an environment's current package list into requirements.txt. If any new library is installed, make sure to run this command to store library version.
 
 ## `uvicorn main:app --reload`
 
 Runs the code inside the backend folder in order to start up the backend server.
 
 Make sure to run this before front-end server `yarn start` to fully functionable.
 
+## `How to host backend server on deta space`
```

Comment:
Put these steps under `docs/` as discussed. Normal developer don't generally need to deploy  and keep README.md slim withh dev set-up only
From Slack,
> Create a README/deployments.md  inside docs directory

Reply:
Addressed in 5a536fedc1e8c3c70df805286afc4ba8e130640e